PathState state
float64[16] sigma
float64[16] lambda
float64[16] covariance